examples/misc/test/language_tour/async_test.dart

```diff
@@ -0,0 +1,41 @@
+// ignore_for_file: type_annotate_public_apis
+import 'dart:async';
+
+import 'package:test/test.dart';
+
+void main() {
+  test('syncGenerator', () {
+    // #docregion sync-generator
+    Iterable naturalsTo(n) sync* {
+      int k = 0;
+      while (k < n) yield k++;
+    }
+
+    // #enddocregion sync-generator
+    expect(naturalsTo(3), [0, 1, 2]);
```

Comment:
You'll want to move the blank line that is at the end of the code excerpt and place it _after_ the `#enddocregion sync-generator` tag.